src/app/components/core/user-management/user-profile/user-profile.component.ts

```diff
@@ -158,4 +161,21 @@ export class UserProfileComponent implements OnInit {
           }
         });
   }
+
+  update() {
+    this.ngProgress.start();
+    this.userService.updateUser(this.userRecord).
+      subscribe(
+        () => {
+          this.ngProgress.done();
+          this.alertService.success('User updated successfully');
```

Comment:
what are we getting from API?

Reply:
It is like-
`{"user_info": {"id": 1, "uname": "admin", "role_id": 1, "access_method": "any", "real_name": "Admin user", "description": "admin user"}}`

Reply:
OK